src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -161,6 +166,10 @@ protected FileMonitoringController doLaunch(FilePath workspace, Launcher launche
         /** @see FileMonitoringTask#charset */
         private @CheckForNull String charset;
 
+        protected String getCharset() {
```

Comment:
Ditto.
```suggestion
        String getCharset() {
```
(or just remove `private` from the field)

Reply:
Done.